.github/workflows/release.yml

```diff
@@ -149,6 +150,7 @@ jobs:
             -DTI_WITH_CC:BOOL=OFF
             -DTI_WITH_VULKAN:BOOL=OFF
             -DTI_BUILD_TESTS:BOOL=ON
+            -DTI_WITH_BACKTRACE:BOOL=ON
```

Comment:
hmmm I'd assume we only want to turn this on for nightly releases but not for official releases?

Reply:
Ah my bad, thanks!